src/Components/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import { Wifi, BatteryFull, Reception4 } from 'react-bootstrap-icons'; // Icons
+
+const Header = () => {
+  const [time, setTime] = useState('');
+
+  useEffect(() => {
```

Comment:
these is wrong just no need to use function for time just use date format for today in html